src/Microsoft.Framework.Runtime/DependencyManagement/LockFileFormat.cs

```diff
@@ -17,91 +16,68 @@ public class LockFileFormat
         public const int Version = -9996;
         public const string LockFileName = "project.lock.json";
 
+        private string _currentLockFilePath;
```

Comment:
This is pretty gross, I'd get rid of it. We don't call this from multiple threads but it looks kinda messy.